toast-tk-dao/src/main/java/io/toast/tk/dao/service/dao/access/repository/RepositoryDaoService.java

```diff
@@ -67,7 +68,9 @@ public boolean saveRepoAsJson(
 		final String jsonRepo
 	) {
 		final GsonBuilder gson = new GsonBuilder();
-		final Type typeOfT = new TypeToken<Collection<RepositoryImpl>>() {private static final long serialVersionUID = 1L;}.getType();
+		final Type typeOfT = new TypeToken<Collection<RepositoryImpl>>() {
+									private static final long serialVersionUID = 1L;}

```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this closing curly brace to the next line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.talanlabs.com/coding_rules#rule_key=squid%3ARightCurlyBraceStartLineCheck)